src/Admin/Settings.php

```
@@ -10,332 +10,334 @@
+    }
+
+    /**
+     * Register all settings.
+     */
+    public function register_settings() {
+        if ( $this->active_tab !== self::GDPRESS_ADMIN_SECTION_MANAGE && $this->active_tab !== self::GDPRESS_ADMIN_SECTION_HELP ) {
+            $this->active_tab = self::GDPRESS_ADMIN_SECTION_MANAGE;
+        }
+
+        foreach ( $this->get_settings() as $value ) {
+            register_setting(
```

PHPStan (2.1.44)

[error] Function register_setting not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)